### ISSUE TITLE
Reference error while fastboot testing

### ISSUE THREAD

**Issue by bobisjan** [cibernox/ember-basic-dropdown#833]
When running tests using `ember-cli-fastboot-testing` the `BasicDropdownWormholeComponent` throws following error

```
Error occurred:

- While rendering:
  -top-level
    application
      BasicDropdownWormholeComponent


There was an error running your app in fastboot. More info about the error: 
 ReferenceError: document is not defined
    at BasicDropdownWormholeComponent.get getDestinationId
```

https://github.com/cibernox/ember-basic-dropdown/blob/master/ember-basic-dropdown/src/components/basic-dropdown-wormhole.ts#L24

It is easy to guard rendering while fastboot testing in application template, but should be this supported out-of-the-box? Thanks

**Comment by mkszepp** [comment]
Hi @bobisjan thanks for reporting... i have added a check if document exists, so it should work while `ember-cli-fastboot-testing`

**Comment by bobisjan** [comment]
@mkszepp thanks for quick reply, I've tested with `v8.0.2`, but the error is still present 😢 

The referenced PR uses `!document`, which throws the same error. Should the fix look something like `typeof document === 'undefined'` instead?

**Comment by mkszepp** [comment]
> @mkszepp thanks for quick reply, I've tested with `v8.0.2`, but the error is still present 😢
> 
> The referenced PR uses `!document`, which throws the same error. Should the fix look something like `typeof document === 'undefined'` instead?

ups... true... sry i fix it asap

**Comment by mkszepp** [comment]
@bobisjan it should be fixed in `v8.0.3`. Let me know if it doesn't work (i have tested shortly with docs page and it seem that the check is now correct)

**Comment by bobisjan** [comment]
@mkszepp tests are green 💚, thanks for the quick fix 🚀
